solidstart-tanstackquery-tailwind-modules/src/components/RepoHeader/tabList.tsx

```diff
@@ -0,0 +1,33 @@
+import { mergeProps } from 'solid-js';
+import { PullRequestIcon, InformationCircleIcon, CodeIcon } from '../Icons';
+
+type CreateTabListProps = {
+  issueCount: number;
+  pullRequestCount: number;
+};
+
+export function createTabList(props: CreateTabListProps) {
```

Comment:
Let's change it in a const in order to add the return type:

```suggestion
export const createTabList = (props: CreateTabListProps): Tabs[] => {
```